docs/accelerator/README.md

```diff
@@ -0,0 +1,66 @@
+# Landing Zone Accelerator
+
+## Background
+
+Collie and its Collie Hub, with all of its available kit modules, are open-source and fully usable for free.
+Anyone can use Collie and the Collie Hub to build and manage their cloud foundation and their landing zones.
+
+Collie and the Collie Hub are offered by [meshcloud](https://meshcloud.io/en/), which is a technology startup
+that offers a Cloud Foundation Platform as a Software-as-a-Service.
```

Comment:
```suggestion
Collie and Collie Hub are offered by [meshcloud](https://meshcloud.io/en/), which is a technology startup
that offers a Cloud Foundation Platform as a Software-as-a-Service.
```

Reply:
d: The part "offers a Cloud Foundation Platform as a Software-as-a-Service" in this context is not really relevant. meshStack does not play a role in the LZ Accelerator. I say we leave this part out to avoid confusion.